### PR TITLE
feat(sdks): upgrade to fb-sdk-16.0.1 

### PR DESCRIPTION
**Pull request by kesha-antonov**
Based on https://github.com/thebergamo/react-native-fbsdk-next/pull/408 and https://github.com/thebergamo/react-native-fbsdk-next/pull/339 but more complete with updates in Readme

### CODE REVIEW

**Comment by Fischaela** [comment]
I would love to see this merged. <3

**Comment by mouessam** [comment]
+1,  any update regarding this point?

File: README.md [hunk 1]
```diff
@@ -121,17 +121,22 @@ Before you can run the project, follow the [Getting Started Guide](https://devel
...
+Follow ***steps 2, 3 and 4*** in the [Getting Started Guide](https://developers.facebook.com/docs/ios/use-cocoapods) for Facebook SDK for iOS.
 
 **NOTE:** The above link (Step 3 and 4) contains Swift code instead of Objective-C which is inconvenient since `react-native` ecosystem still relies
    on Objective-C. To make it work in Objective-C you need to do the following in `/ios/PROJECT/AppDelegate.m`:
-   1. Add `#import <FBSDKCoreKit/FBSDKCoreKit-swift.h>`
+   1. Add
+   ```objc
+   #import <AuthenticationServices/AuthenticationServices.h>
+   #import <SafariServices/SafariServices.h>
+   #import <FBSDKCoreKit/FBSDKCoreKit-swift.h>`
```
Comment (lukasgjetting) [inline README.md:132]: Typo: backtick

**Comment by jaredstrong89** [comment]
+1, keeping an eye on this one

**Review by lukasgjetting — COMMENTED** [review]
Thanks for setting this up! 👍  Hoping it will be merged soon. In the meantime, a few comments that may (or may not) be of use.

- While the removal of trailing whitespace is an improvement, it might be better to minimize the diff by undoing those changes? Not sure...
- Newest version is now 16.1.0, do you know if it would require any changes other than to `FBSDKVersion` to support it?

**Comment by kesha-antonov** [comment]
> Thanks for setting this up! 👍 Hoping it will be merged soon. In the meantime, a few comments that may (or may not) be of use.
> 
> * While the removal of trailing whitespace is an improvement, it might be better to minimize the diff by undoing those changes? Not sure...
> * Newest version is now 16.1.0, do you know if it would require any changes other than to `FBSDKVersion` to support it?

Hi
Let's see what the maintainers will say

**Comment by zdnk** [comment]
Any update?

**Comment by zdnk** [comment]
Really waiting for this to get merged and released so we can leverage the AEM AutoSetup in our Expo app and start using advantage ads on Meta. Let me know if I can help in any way.

**Comment by ps-samclarke** [comment]
Meta have a compliance deadline against this SDK of August 1st, is it possible to see this merged ASAP so we can avoid any issues?

**Comment by noma4i** [comment]
Hi! Any updates to merge this?

**Comment by mikehardy** [comment]
Superceded by #441 - the updates are happening though!

**Pull request closed without merging**